crates/ibc-core/ics04-channel/Cargo.toml

```
@@ -0,0 +1,84 @@
+[package]
+name         = "ibc-core-channel"
+version      = { workspace = true }
+authors      = { workspace = true }
+edition      = { workspace = true }
+rust-version = { workspace = true }
+license      = { workspace = true }
+repository   = { workspace = true }
+keywords     = ["blockchain", "cosmos", "ibc", "channel"]
+readme       = "./../../README.md"
+description  = """
+    Maintained by `ibc-rs`, contains the implementation of the ICS-04 Channel & Packet Semantics and 
```

```suggestion
    Maintained by `ibc-rs`, contains the implementation of the ICS-04 channel & packet semantics and 
```

If you want to keep this capitalized, do the same for `ics02-client`'s `Cargo.toml` :slightly_smiling_face: